File: problems/F-soccer-passes/solution/SoccerPasses.java
import java.util.Scanner;

public class SoccerPasses {

    // Function to calculate the distance between two points
    public static double calculateDistance(int x1, int y1, int x2, int y2) {
        return Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Parse the input for Player A's coordinates
        int x1 = scanner.nextInt();
        int y1 = scanner.nextInt();

        // Parse the input for Player B's coordinates
        int x2 = scanner.nextInt();
        int y2 = scanner.nextInt();

        // Calculate the distance
        double distance = calculateDistance(x1, y1, x2, y2);

        // Print the result rounded to two decimal places
        System.out.printf("%.2f%n", distance);

        scanner.close();
    }
}

File: problems/F-soccer-passes/solution/soccer-passes.py
import math

def calculate_distance(x1, y1, x2, y2):
    """Calculate the distance between two points (x1, y1) and (x2, y2)."""
    return math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)

# Parse the input
x1, y1 = map(int, input().split())
x2, y2 = map(int, input().split())

# Calculate the distance
distance = calculate_distance(x1, y1, x2, y2)

# Print the result rounded to two decimal places
print(f"{distance:.2f}")